### DISCUSSION TITLE
[Enhancement] [table] table-tree 懒加载如何新增子节点和移动位置

### DISCUSSION THREAD

**Discussion by lizhong43403** [element-plus/element-plus#12948] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

1. 可否新增树形表格的子项创建的实例
2. 可否支持树形表格的子项位置调整，即调整单项所在的父级所在位置。或子项调整成顶层分类，或顶层分类移动至子项

### Additional comments

_(empty)_

**Comment by elarbor** [comment]
[展开行](https://element-plus.org/zh-CN/component/table.html#%E5%B1%95%E5%BC%80%E8%A1%8C)怎么做懒加载啊？不是树形数据
![动画](https://github.com/user-attachments/assets/338858fd-1683-4431-9b22-020fa9cb68a3)
